README.md

```diff
@@ -333,3 +333,79 @@ $ npm run lint
 ## Github Actions
 
 All pushes to this repo will be checked with `npm test` and `npm lint`.
+
+## Git Workflow
+
+Our branches (in order or stability are):
+
+| Branch     | Environment | AWS Account   | Link To Application                                                               |
+|:-----------|:------------|:--------------|:----------------------------------------------------------------------------------|
+| main       | development |               | [localhost:3000](http://localhost:3000)                                           |
+| qa         | qa          | nypl-dams-dev | [https://qa-dcfrontend.nypl.org/](https://qa-dcfrontend.nypl.org/)                |
+| production | production  | nypl-dams-prod| [https://dcfrontend.nypl.org/](https://dcfrontend.nypl.org/)                      |
+
+## Contributing
+
+1.  Feature branches are cut from `main` using 
+
+    `git checkout -b feature/DR-123/feature-name`.
+
+2.  Add your changes to the CHANGELOG.md file under `## [Unreleased]`. See [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) for formatting docs.
+3.  Create a pull request of the `feature` branch _into_ `main`. 
+4.  After a feature branch has approved accessibility review and VQA, merge `feature` branch into `main`, then merge `main` branch into `qa` for testing.
+5.  After QA has tested and approved cut a `release` branch off of `qa` using our Release Naming Strategy. Please see Release Naming Strategy for details on how we plan to number our releases during the initial rollout of this project.
+
+   `git checkout -b release/x.y.z`.
+
+(see tagged releases or changelog for last version)
+
+5. Update CHANGELOG.md in release branch by moving updates from unreleased into the new release section.
+
+    ie. `## [2.4.13] - 04-21-2023`
+
+6. Commit and push changes to release branch.
+8. Merge `release` branch to `production` and push the changes. This will deploy through Travis if all goes well.
+10. Immediately after merging release to production, finish the release and backmerge release to `qa` and `main`
+    
+    `$ git tag -a x.y.z` 
+    
+    `$ git push origin --tags`
+
+### Release Naming Strategy 
+This project is in the beginning phases of it's development. We will use a custom versioning system inspired by Semantic Versioning while we migrate the functionality of the digitalreadingroom app to this app. Once all of the initial Phase (as defined in the [Product Brief](https://docs.google.com/document/d/187LA6VjOaDdXA6xRZYv_gqgNzJcAyuSPIZoenVH2L3w)) are complete, this repo will use [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+### Versioning system:
+MAJOR: all of the Phases are complete 
+MINOR: phase, set of features required to meet the mvp of a Phase as defined in the Product Brief or TAD.
+PATCH: feature or hotfix that supports a component or section of the mvp of a Phase
+
+Format: MAJOR.MINOR.PATCH
+
+### Summary of Phases (to be removed later?)
```

Comment:
This is good dev docs to keep. It might be better in its own file that you can link to from here but it doesn't have to be done right now.